docs/index.md

```diff
@@ -12,6 +12,7 @@ The Veteran-facing Services Platform (VSP) is the common infrastructure, technic
 * DSVA Slack
     * [#platform-team](https://dsva.slack.com/messages/CJRQ85PQB), for general discussion
     * [#vfs-platform-support](https://dsva.slack.com/messages/CBU0KDSB1), for platform support
+    * [#vsp-triage](https://dsva.slack.com/messages/CK1FA11H8), for Triage Issues
```

Comment:
(a) for a general audience --- what constitutes "Triage Issues"? I don't think that's self-evident

(b) does the Triage team want people dropping into #vsp-triage as a channel for inbound issues? seems like that's more for the team's own communication, idk if we want to mix those two or not.

Reply:
a) I think how you changed better describes it " Submit a non-critical issue to be triaged "  Do you think that is sufficient or do we need to expand further on that?

b) I went back and forth on this one.  And since you mentioned it, no we probably don't want that.  Too many cooks (aka channels) in the kitchen for that so we can remove it.